### PR TITLE
IAV-2209: Update styleguide with form elements

### PR DESCRIPTION
**Pull request by kallevu**
- Adds `:active` and `:hover` styles for form elements.
- Unifies colors.
- Adds Chosen.js library for select element.

### CODE REVIEW

**Comment by lauriii** [comment]
screenshot from 2017-03-24 18-45-39
Should we add some padding here?

screenshot from 2017-03-24 18-45-51
Should we add some margin between buttons?

screenshot from 2017-03-24 18-45-43
Maybe some nicer test content would be useful here (this is a public website)

**Comment by juhog** [comment]
Textarea padding, see this commit: https://github.com/druidfi/aava/pull/43/commits/d5a4d2644bb2c27ba99adc3a3783675c51a87967

**Comment by juhog** [comment]
I fixed the textarea padding issue (styleguide commits https://github.com/LaakarikeskusAava/styleguide/pull/16/commits/2e7fcbc87b9dbdaa7f9adb7777051b3ef4b875c0 and https://github.com/LaakarikeskusAava/styleguide/pull/16/commits/c09e6906315d095ed2886e067c429aa3341e4313).

**Comment by juhog** [comment]
Form element font-sizes are not set and the font-size depends on browser (chrome 11px in the screen below).

I'm gonna fix this.

screen shot 2017-03-26 at 09 42 16

**Comment by juhog** [comment]
Buttons now have margin-right 0.5em.

screen shot 2017-03-26 at 09 59 41

**Comment by juhog** [comment]
Radio button options are now "Option A" and "Option B": https://github.com/LaakarikeskusAava/styleguide/pull/16/commits/c5fd92fabdb500db95d8da23984ca05109ea6d8a

**Review by juhog — CHANGES_REQUESTED** [review]
Header button (Online ajanvaraus) seems to be missing. Other than that looks good.

File: styles/components/buttons/buttons.scss [hunk 1]
```diff
@@ -80,3 +107,30 @@ $icon_width: 10px;
...
+  margin-bottom: 10px;
+  padding: 10px 20px;
+  .label {
+    color: $black;
+    display: inline;
+    font-weight: normal;
+    padding-right: 0.5em;
+  }
+}
+@media screen and (min-width: 769px) {
```
Comment (juhog) [inline styles/components/buttons/buttons.scss:127]: @lauriii Miten tehään responsive tyylien kanssa styleguidessa? Tässä ensimmäinen tilanne missä on tulossa media-querya styleguideen.

**Review by juhog — APPROVED** [review]

**Pull request merged**